plugins/backend/local/localInterface.vala

```diff
@@ -393,9 +393,7 @@ public class FeedReader.localInterface : Peas.ExtensionBase, FeedServerInterface
 		{
 			string cID = createCategory(newCatName, null);
 			var cat = new Category(cID, newCatName, 0, 99, CategoryID.MASTER.to_string(), 1);
-			var list = new Gee.ArrayList<Category>();
-			list.add(cat);
-			m_db_write.write_categories(list);
+			m_db_write.write_categories(ListUtils.single(cat));
```

Comment:
Thanks for fixing these too! I didn't realize this was existing code you had copied.